mender/CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Mender Helm chart
 
-## master
+## 5.5.0
 * Fix mongodb uri creation when using the mongodb subchart and replicast architecture
+* Added customEnv option to set default or per-service custom env variables
+* Added generic `storage_proxy` service, that could
+  work for both minio and Amazon S3, and it's going to replace the `api_gateway.minio` configuration.
+* **Deprecations**:
```

Comment:
I chose a soft deprecation: `api_gateway.minio` is still enabled (and default choice) but it's considered deprecated.